src/rez/rex_bindings.py

```diff
@@ -243,6 +263,10 @@ def commands():
     elif isinstance(obj, VersionBinding):
         range2 = VersionRange(str(obj))
 
+    # eg 'if intersects(ephemerals.get('foo.cli'), ...)'
```

Comment:
This is hand-holding a bit too much imo. Better to let it fail with the RuntimeError (which should really be a ValueError, damn..)